### ISSUE TITLE
DO_CORR data from 2016-08-01 to 2016-09-14 in HB and CR are swapped/mislabeled

### ISSUE THREAD

**Issue by brittnibertolet** [MFEh2o/db#40]
Raw DO miniDOT data on Onedrive confirms that the serial# of the miniDot switched between HB and CR during the last download. The temp chain data is correct. Only the miniDot data got switched.

**Comment by kaijagahm** [comment]
@brittnibertolet I'm just starting to work with the sensor database. Can you explain what the "temp chain data" is?

**Comment by kaijagahm** [comment]
Just to confirm that I understood this and did the fix correctly:
Changed all rows in DO_CORR with dates between 2016-08-01 and lakeID == HB to lakeID == CR
Changed all rows in DO_CORR with dates between 2016-08-01 and lakeID == CR to lakeID == HB

@brittnibertolet does this look right? Do I need to change anything in DO_RAW,  or only DO_CORR? I just blindly followed your description of the problem here; not sure how to tell for myself which samples are wrong. How did you find them in the raw data?

**Comment by brittnibertolet** [comment]
Yes, you interpreted correctly and made the correct fix! Thanks! DO_RAW should also be switched in the same way. 

I learned of this issue when diving into GPP data, which is derived from DO data. You can see for the period between 2016-08-01 and 2016-09-14 in HB and CR (between the red lines in the plot), the data for HB and CR are unlike anything seen prior or after. It looked to me that the sensors were mislabeled during download. I went to the raw data (pre B3, which is stored on OneDrive), and confirmed that the serial numbers for the sensors associated with the data were switched for the time period.
[GPP.pdf](https://github.com/MFEh2o/db/files/5257259/GPP.pdf)

**Comment by kaijagahm** [comment]
Great, thanks for the very clear explanation, @brittnibertolet! I'll make the switch in the raw data too, and call this good.

**Comment by kaijagahm** [comment]
Need help on this one figuring out how to actually do a database update for the sensorDB.

Also need to go through and actually implement this fix in the raw data.

**Comment by kaijagahm** [comment]
From 12/15 meeting: take a look at Katie's instructional document for the sensor database.

**Comment by kaijagahm** [comment]
Check https://app.box.com/file/730074958857 (DatabaseManagementInstructions_UsingR_20200526.docx). But at the 1/5/2021 meeting, we decided that we're going to push sensor database updates to the back burner in favor of continuing to work on main database issues, and in favor of big-picture database tools like fish entry and checks.

**Comment by kaijagahm** [comment]
This issue has a script started. See 'gh040_DO_sensor_fix.R' in my archive.
